docs(getting_started): typo's, simplified and correct language (#391)
arcus-azure/arcus.testing: docs/preview/02-getting-started.md

**Welcome to Arcus Testing! 🎉**

This page is dedicated to be used as a walkthrough on how to integrate Arcus Testing in new and existing projects.
Arcus Testing is an umbrella term for a set of `Arcus.Testing.*` NuGet packages that kick-start your code testing. 

## The basics

> The packages in the Arcus Testing space are split up in these main categories:

Reviewer: ```suggestion
The packages in the Arcus Testing space are split into the following categories:
```
